include/mxnet/operator.h

```diff
@@ -287,6 +287,17 @@ class OperatorProperty {
    */
   virtual Operator* CreateOperator(Context ctx) const = 0;
   /*!
+   * \brief Create a Operator on specific context and input type
+   */
```

Comment:
Maybe document that eventually we will deprecate the CreateOperator function and move to CreateOpEx